packages/pillarbox-playlist/src/components/modal/pillarbox-previous-item-button.js

```diff
@@ -0,0 +1,51 @@
+import videojs from 'video.js';
+
+/**
+ * @ignore
+ * @type {typeof import('video.js/dist/types/button').default}
+ */
+const Button = videojs.getComponent('Button');
+
+/**
+ * The previous item button for the playlist ui. When clicked moves to the
+ * previous item in the playlist.
+ */
+class PillarboxPlaylistPreviousItemButton extends Button {
+  constructor(player, options) {
+    super(player, options);
+    this.handleLanguagechange();
+    this.setIcon('previous-item');
+  }
+
+  ready() {
+    this.$('.vjs-icon-placeholder').classList.toggle(`vjs-icon-previous-item`, true);
+  }
+
+  /**
+   * Get the playlist instance associated with the player.
+   *
+   * @returns {import('pillarbox-playlist.js').default} The playlist instance.
+   */
+  playlist() {
+    return this.player().pillarboxPlaylist();
+  }
+
+  /**
+   * Handles the click event on the button.
+   *
+   * @param {Event} event - The click event.
+   */
+  handleClick(event) {
+    super.handleClick(event);
+    this.playlist().previous();
+  }
+
+  /**
+   * Handles the language change event to update the control text.
+   */
+  handleLanguagechange() {
+    this.controlText(this.localize('Previous Item'));
+  }
```

Comment:
Two things:

1. FYI `controlText` calls the localize method internally, so there is not need to call `this.localize`
2. could we remove the `handleLanguagechange` method and set the `controlText` via an option in the constructor ? The reason is that it breaks video.js API so a developer can't change the default control text.
```javascript
player.pillarboxPlaylistMenuDialog.getChild('PillarboxPlaylistControls').children()[0].controlText('Mute');
player.controlBar.playToggle.controlText('Mute');

player.language("it");

player.pillarboxPlaylistMenuDialog.getChild('PillarboxPlaylistControls').children()[0].controlText() // value is reset
player.controlBar.playToggle.controlText(); // value is kept
```

Reply:
Done.